charts/apisix/Chart.yaml

```
@@ -31,12 +31,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.8.1
+version: 0.8.2
```

Sorry, can you please bump this to 0.8.3?  https://github.com/apache/apisix-helm-chart/pull/233/ has been merged.  It has modified this to 0.8.2.
```suggestion
version: 0.8.3
```

---

Updated